source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: "Customize the S3-BOX with your own images"
+product_name: ESP32-S3-BOX
+device_name_entry: ESP32-S3-BOX
+config_link: /voice_control/thirteen-usd-voice-remote/#to-delete-the-atom-echo-configuration-from-esphome
+---
+
+This tutorial will show you how to replace the Home Assistant status images on the ESP32-S3-BOX by custom images. 
```

Comment:
Include a video of the Frenck box here.

Reply:
@balloob Can you past the youtube link once uploaded? We don't have the rights